plugins/obs-outputs/WebRTCStream.cpp

```diff
@@ -273,17 +274,8 @@ bool WebRTCStream::start(Type type)
     rtc::scoped_refptr<webrtc::VideoTrackSourceInterface> videoSource = videoCapturer->getNativeVideoSource();
     video_track = nativeFactory->CreateVideoTrack("video", videoSource);
     videoCapturer->start();
-    //Add track to the stream
-    stream->AddTrack(video_track);
-  
-    //Add the stream to the peer connection
-    if (!pc->AddStream(stream))
-    {
-        //Log
-        error("Adding stream to PeerConnection failed");
-        //Error
-        return false;
-    }
+    //Add video track to peer connection
+    pc->AddTrack(video_track, {"obs"});
```

Comment:
It s a pity that we lost the capacity to capture an error. Isn't there an equivalent error message with pc->addTrack ?

Reply:
pc->AddTrack has a return type of: RTCErrorOr<rtc::scoped_refptr<RtpSenderInterface>>